### PR TITLE
feat(third-party-contrib): update entries

### PR DESCRIPTION
**Pull request by naiduarvind**
Signed-off-by: Arvind Naidu <arvind@thebility.engineer>

Included Github Actions for buildevents and honeymarker.

### CODE REVIEW

**Comment by naiduarvind** [comment]
@kvrhdn - added your buildevents entry here.

File: README.md [hunk 1]
```diff
@@ -34,6 +34,6 @@ If you'd like to add your own project, please file a pull request doing so.
 | honeycomb-logger | Nginx proxy that logs to stdout events sent to Honeycomb. | https://github.com/darklang/honeycomb-logger |
 | cl-honeycomb | Common Lisp client for sending events | https://github.com/franzinc/cl-honeycomb/ |
 | terraform-provider-honeycombio | Terraform provider for managing Honeycomb resources | https://github.com/kvrhdn/terraform-provider-honeycombio 
-| honeycomb-create-marker | Github Action for creating Honeycomb Markers | https://github.com/beauknowssoftware/honeycomb-create-marker
-| honeycomb-update-marker | Github Action for updating Honeycomb Markers | https://github.com/beauknowssoftware/honeycomb-update-marker
+| gha-buildevents | Add observability to your GitHub Actions workflows | https://github.com/kvrhdn/gha-buildevents
```
Comment (MikeGoldsmith) [inline README.md:37]:
```suggestion
| gha-buildevents | Trace GitHub Action workflows and send to Honeycomb | https://github.com/kvrhdn/gha-buildevents
```

**Review by MikeGoldsmith — APPROVED** [review]
Looks good, thanks for the update @naiduarvind.

I've just made one suggestion for @kvrhdn's `gha-buildevents` description. Let me know what you think.

File: README.md (hunk 1 above)
Comment (kvrhdn) [reply README.md:37]: LGTM. I agree that this is more appropriate 👍🏻

**Comment by kvrhdn** [comment]
Thanks for including my action @naiduarvind, forgot about adding it to this repo 🙂

**Comment by MikeGoldsmith** [comment]
Thanks @naiduarvind & @kvrhdn 👍

**Pull request merged**